recipes/jupyterlab-fasta/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "jupyterlab-fasta" %}
+{% set version = "3.1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab-fasta-{{ version }}.tar.gz
+  sha256: 0323d18d6ce9dbbe381aaa1d94d362ea468699eabbcebd55c397ce482c989b7f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - jupyter-packaging >=0.7.9,<0.8.0
+    - pip
+    - python >=3.6
+  run:
+    - jupyterlab ~=3.0
+    - python >=3.6
+
+test:
+  commands:
+    - pip check
+    - test -f "${PREFIX}/share/jupyter/labextensions/@jupyterlab/fasta-extension/package.json"                                    # [unix]
+    - if exist %PREFIX%\\share\\jupyter\\labextensions\\@jupyterlab\\fasta-extension\\package.json (exit 0) else (exit 1)         # [win]
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jupyterlab/jupyter-renderers
+  summary: Fasta renderer for JupyterLab
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jtpio
+    - jasongrout
```

Comment:
cc @jasongrout let me know if it's ok to have you as maintainer here (and in the other recipes)

Reply:
Please add me as a maintainer as well

Reply:
Done, thanks!